src/main/java/woohakdong/server/common/config/SecurityConfig.java

```diff
@@ -2,18 +2,30 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import woohakdong.server.common.security.jwt.JWTFilter;
+import woohakdong.server.common.security.jwt.JWTUtil;
 
 @Configuration
 @EnableWebSecurity
 public class SecurityConfig {
+    private final JWTUtil jwtUtil;
+
+    public SecurityConfig(JWTUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
```

Comment:
친절한 주석 감사합니다.

```suggestion
@Configuration
@EnableWebSecurity
@RequiredArgsConstructor
public class SecurityConfig {
    private final JWTFilter jwtFilter
```

혹시 이렇게 수정한 뒤에, JWTFilter도 component로 처리하는 것이 어떨까요?